tests/test_api_handler.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import asyncio
+from Modules.api_handler import *
+
+
+class APIHandlerTest(unittest.TestCase):
+    handler = APIHandler(hostname="dev.api.fuelrats.com", tls=True)
```

Comment:
Test cases have a setUp() method for this, giving better output if the constructor should fail